Atomic/backends/_docker_errors.py

```diff
@@ -0,0 +1,7 @@
+class DockerObjectNotFound(ValueError):
```

Comment:
OK, one other question, why are you using the _NAME.py syntax for these python objects?